app/views/result_comments/_comment.html.erb

```diff
@@ -15,25 +15,21 @@
       </a>
       <ul class="dropdown-menu dropdown-menu-fixed" aria-labelledby="comment-<%= comment.id %>-dropdown">
         <li class="dropdown-header"><%= I18n.t('comments.options_dropdown.header') %></li>
-        <% if can_edit_result_comment_in_module(comment) %>
-          <li>
-            <a href="#"
-               data-action="edit-comment"
-               data-url="<%= edit_result_result_comment_path(comment.result, comment, format: :json) %>">
-              <%= t('comments.options_dropdown.edit') %>
-            </a>
-          </li>
-        <% end %>
-        <% if can_delete_result_comment_in_module(comment) %>
-          <li>
-            <a href="#"
-               data-action="delete-comment"
-               data-url="<%= result_result_comment_path(comment.result, comment, format: :json) %>"
-               data-confirm-message="<%= t('comments.delete_confirm') %>">
-              <%= t('comments.options_dropdown.delete') %>
-            </a>
-          </li>
-        <% end %>
+        <li>
+          <a href="#"
```

Comment:
can you disable turbolinks here this may cause some problems.

Reply:
Sure, buddy!